.github/CONTRIBUTING.md

```diff
@@ -76,11 +76,14 @@ Check the [cakephp-codesniffer](https://github.com/cakephp/cakephp-codesniffer)
 repository to set up the CakePHP standard. The [README](https://github.com/cakephp/cakephp-codesniffer/blob/master/README.md) contains installation info
 for the sniff and phpcs.
 
-To run static analysis tools [PHPStan](https://github.com/phpstan/phpstan) and [Psalm](https://github.com/vimeo/psalm) you first have to install the additional packages via:
+To run static analysis tools [PHPStan](https://github.com/phpstan/phpstan) and [Psalm](https://github.com/vimeo/psalm) you first have to install the additional packages via [phive](https://phar.io/).
 
-    composer stan-setup
+    phive install phpstan@1.9
+    phive install psalm@5.4
```

Comment:
Updating this version number will be a forgotten chore. psalm releases new minor every few weeks.

Reply:
replaced it with a placeholder